ftests/nuxeo-jsf-ui-webdriver-tests/src/test/java/org/nuxeo/ftest/cap/ITSearchTabTest.java

```diff
@@ -215,18 +213,6 @@ public void testSearch() throws UserNotConnectedException, IOException {
         resultPanelSubPage = searchPage.getSearchResultsSubPage();
         assertEquals(nbCurrentDoc, resultPanelSubPage.getNumberOfDocumentInCurrentPage());
 
-        // Test Collections Widget
```

Comment:
I think this portion of the test is still useful to test search by collection membership, independently from My Favorites.

Reply:
Yes, as far as I was able to understand, no other collection was created and that's the reason why we were pointing to "My Favorites"